packages/dashboard/src/pages/Wallets/components/FungibleTokenTable/index.tsx

```diff
@@ -95,10 +95,10 @@ export const FungibleTokenTable = memo<TokenTableProps>(({ selectedChainId }) =>
         openSwapDialog({
             open: true,
             traderProps: {
-                coin: {
+                defaultFromCoin: {
                     id: token.id,
-                    name: token.name ?? '',
-                    symbol: token.symbol ?? '',
+                    name: token.name || '',
```

Comment:
This is an abuse of `??`, what if token.name is `0` at runtime?

Reply:
Why a token's name will be `0`?

Reply:
> Why a token's name will be `0`?

Just as an example. And well, runtime can be unpredictable.